src/commands/generate/fromTemplate.ts

```diff
@@ -30,6 +30,29 @@ interface ParsedFlags {
   mapBaseUrlToFolder: IMapBaseUrlToFlag
 }
 
+const templatesNotSupportingV3: Record<string, string> = {
+  '@asyncapi/minimaltemplate': 'some link', // For testing purpose
+  '@asyncapi/html-template': 'https://github.com/asyncapi/html-template/issues/430',
+  '@asyncapi/dotnet-nats-template': 'https://github.com/asyncapi/dotnet-nats-template/issues/384',
+  '@asyncapi/ts-nats-template': 'https://github.com/asyncapi/ts-nats-template/issues/545',
+  '@asyncapi/python-paho-template': 'https://github.com/asyncapi/python-paho-template/issues/189',
+  '@asyncapi/nodejs-ws-template': 'https://github.com/asyncapi/nodejs-ws-template/issues/294',
+  '@asyncapi/java-spring-cloud-stream-template': 'https://github.com/asyncapi/java-spring-cloud-stream-template/issues/336',
+  '@asyncapi/go-watermill-template': 'https://github.com/asyncapi/go-watermill-template/issues/243',
+  '@asyncapi/java-spring-template': 'https://github.com/asyncapi/java-spring-template/issues/308',
+  '@asyncapi/markdown-template': 'https://github.com/asyncapi/markdown-template/issues/341'
```

Comment:
we are missing:
- @asyncapi/nodejs-template
- @asyncapi/java-template
- @asyncapi/php-template

Reply:
I have missed php-template completely, nice catch 👍

Reply:
Added 👍